### ISSUE TITLE
BT-195 (Unpublished Identifier); Description; decision on publication

### ISSUE THREAD

**Issue by tinne** [eForms/eForms#70]
The description to this field says ”Identifier of the business term that should not be published.“

While we understand that not every business term allows for being left private and that a plausible justification must be given in every single case, the process of how it is decided what is published and what is not is not clear from this wording.

A clear version would be ”Identifier of the business term that must not be published,“ where every rejecting (EUR-OP would have reasons to publish the term value nevertheless) results in an error, effectively giving the tenderer to decide on any such rejection and not having the value published because someone else decided that a should is only a should.

- What is the decision process going to be like?
- Which terms are allowed to be left unpublished?
- Am I right to expect that BG-8 (Not Published) itself is never published?

**Comment by JachymHercher** [comment]
You are correct, we will replace the ambiguous "should" for the clear "must". TED will not "second guess" the buyer's decision (except, possibly, random checks that the string in BT-196 (Unpublished Justification Description) is not random characters). 

1) Technically - just marking it in the notice. Legally - fairly complex. The directive's default is the publication of everything. 2014/24/EU Art. 50(4), on which the right to not publish is based, is very vague and as far as I've seen, it's interpretation around the EU is extremely diverse. Having a few court decisions in the future may be necessary to have clarity on what precisely it means. 

2) Any BT in BG-320 (Winning Tender) and BG-137 (Procedure Lot Result) (as long as the non-publication can be legally justified). 

3) We would expect it to be published - the BTs add transparency on the buyer's decision to publish (or not). Also, the BTs themselves do not contain confidential information, so there's no reason to keep them unpublished. (The only BT which theoretically could contain confidential information itself is BT-196 (Unpublished Justification Description), but in practice we would expect that this could be fairly easily avoided by the buyer when writing the justification.)

**Comment by tinne** [comment]
The justification is BT-based and BT-195 is not repeatable?

Why would one have a Not Published BG without an Unpublished Identifier?

Organisation information is restricted if and only if every reference to it is restricted?

**Comment by tinne** [comment]
@JachymHercher writes ”BT-198 should be M, not CM.“

Does this means that all unpublished information must be disclosed some day? Are there any restrictions on this date?

**Comment by JachymHercher** [comment]
Yes. 

In principle there should be (nothing is secret forever), but we haven't discussed for how long. Happy for opinions, source or analogies (e.g. from other areas of law such as trade secrets) which could serve as a basis for setting this number. 

(Very upper boundry could be inspired by government classified information, which seems to be declassified in several countries in [around thirty years](https://en.wikipedia.org/wiki/Declassification). Intuitively, I would expect trade secrets to remain trade secrets for much shorter periods. Matters of national security - not sure.)

The setting of this date should also be justified. As such, I would suggest changing the definition of BT-196 (Justification Description) from "Justification for not publishing some fields." to "Justification for not publishing some fields and for setting the later date at which they can be published."

**Comment by JachymHercher** [comment]
In light of https://github.com/eForms/eForms/issues/70#issuecomment-403518662, we have also changed the descriptions of the BTs from plural to singular.
